doc/release-notes/5.7-release-notes.md

```diff
@@ -0,0 +1,109 @@
+- Administrators and curators can now receive notifications when a dataset is created (Issue #8069, PR #8070)
+- Administrators with large files in their installation can disable the automatic checksum verification process at publish time (Issue #8043, PR #8074)
+
+## Notes for Dataverse Installation Administrators
+
+### Dataset Creation Notifications for Administrators
+
+A new :SendNotificationOnDatasetCreation setting has been added. When true, administrators and curators (those who can publish the dataset) will get an email/notification when a new dataset is created. This makes it easier to track activity in a Dataverse and, for example, allow admins to follow up when users do not publish a new dataset within some period of time.
+
+### Skip Checksum Validation at Publish
+
+When a user requests to publish a dataset, the time taken to complete the publishing process varies based on the dataset/datafile size.
+
+With the additional settings of :DatasetChecksumValidationSizeLimit and :DataFileChecksumValidationSizeLimit, the checksum validation can be skipped while publishing.
+
+If the Dataverse administrator chooses to set these values, it's strongly recommended to have an external auditing system run periodically in order to monitor the integrity of the files in the Dataverse installation.
+
+### Guestbook Response API Update
+
+With this release the Retrieve Guestbook Responses for a Dataverse Collection API will no longer produce a file by default. You may specify an output file by adding a -o $YOURFILENAME to the curl command.
+
+### Dynamic JavaServer Faces Configuration Options
+
+This release includes a new way to easily change JSF settings via MicroProfile Config, especially useful during development.
+See the [development guide on "Debugging"](https://guides.dataverse.org/en/5.7/developers/debugging.html) for more information.
+
+### Enhancements to DDI Metadata Exports
+
+Several changes have been made to the DDI exports to improve support for internationalization and to improve compliance with CESSDA requirements. These changes include:
+
+- Addition of xml:lang attributes specifying the dataset metadata language at the document level and for individual elements such as title and description
+- Specification of controlled vocabulary terms in duplicate elements in multiple languages (in the installation default langauge and, if different, the dataset metadata language)
+
+While these changes are intended to improve harvesting and integration with external systems, they could break existing connections that make assumptions about the elements and attributes that have been changed.
+
+## New JVM Options and DB Settings
+
+- :SendNotificationOnDatasetCreation - A boolean setting that, if true will send an email and notification to users when a Dataset is created. Messages go to those, other than the dataset creator, who have the ability/permission necessary to publish the dataset.
```

Comment:
```suggestion
- :SendNotificationOnDatasetCreation - A boolean setting that, if true will send an email and notification to additional users when a Dataset is created. Messages go to those, other than the dataset creator, who have the ability/permission necessary to publish the dataset.
```

Also, we could probably just simply this to "notification" since notifications are always email and in-app.